lib/plot/falco_plot_progress.m

```diff
@@ -41,13 +41,13 @@
 
             %--Top label
             handles.textbox1 = annotation('textbox', [0.25 0.89 0.5 0.1], ...
-                'String', sprintf('%s: Iteration %d',mp.coro,Itr-1),'Fontsize',32,...
+                'String', sprintf('%.e bw, %s: Iteration %d',mp.FracBW,mp.F3.phaseMaskType,Itr-1),'Fontsize',32,...
```

Comment:
Why is the figure title being changed in the 'HLC' case?

Reply:
This is somewhat weird because if it isn't a phase mask, it will default this variable to saying `vortex` and then show that in plots for the HLC.

Reply:
i didn't mean to keep these changes. plot_progress can be kept as the original falco script. you can ignore any temporary modifcations i may have made here.